keel-sql/src/main/resources/db/changelog/20200923-add-created-at-to-artifact-versions.yml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-created-at-to-artifact-versions
+      author: lpollo
+      changes:
+        - addColumn:
+            tableName: artifact_versions
+            columns:
+              - column:
+                  name: created_at
+                  type: timestamp(6)
+                  constraints:
+                    nullable: true
```

Comment:
Is this nullable because we have old data and no way to back-fill the value, or because it comes from metadata after the initial record is written?

Reply:
The former. We could potentially infer the value from `buildMetadata.completedAt`, but I'm guessing we'll miss some. Could we use epoch time as a default? We could also write a component to fetch the data from Rocket and make the column not nullable in a follow-up PR.